### DISCUSSION TITLE
Error: My project is not starting up.

### DISCUSSION THREAD

**Discussion by DevSammy22** [tailwindlabs/tailwindcss#9815] [General]
My project is not starting up.

This is the link to my repo: https://github.com/DevSammy22/Krypt-Web3-Full-Stack-Project.git
module.exports = {
  purge: ["./src/**/*.{js,jsx,ts,tsx}", "./public/index.html"],
  mode: "jit",
  darkMode: false, //"media", // or 'media' or 'class'

  theme: {
    fontFamily: {
      display: ["Open Sans", "sans-serif"],
      body: ["Open Sans", "sans-serif"],
    },
    extend: {
      screens: {
        mf: "990px",
      },
      keyframes: {
        "slide-in": {
          "0%": {
            "-webkit-transform": "translateX(120%)",
            transform: "translateX(120%)",
          },
          "100%": {
            "-webkit-transform": "translateX(0%)",
            transform: "translateX(0%)",
          },
        },
      },
      animation: {
        "slide-in": "slide-in 0.5s ease-out",
      },
    },
  },
  variants: {
    extend: {},
  },
  plugins: [require("@tailwindcss/forms")],
};

When I run npm run dev I got the picture below
![image](https://user-images.githubusercontent.com/98836006/201447669-8b927d83-78bc-4e53-974b-685c8dbb9e99.png)

**Comment by wongjn** [comment]
This is the error I got:
```
[vite] Internal server error: Failed to resolve import "../utils/constants" from "src/context/TransactionContext.jsx". Does the file exist?
  Plugin: vite:import-analysis
  File: /client/src/context/TransactionContext.jsx:3:45
  16 |  import React, { useEffect, useState } from "react";
  17 |  import { ethers } from "ethers";
  18 |  import { contractABI, contractAddress } from "../utils/constants";
     |                                                ^
  19 |  import { jsxDEV as _jsxDEV } from "react/jsx-dev-runtime";
  20 |  export const TransactionContext = React.createContext();
      at formatError (file:///client/node_modules/vite/dist/node/chunks/dep-c842e491.js:41168:46)
      at TransformContext.error (file:///client/node_modules/vite/dist/node/chunks/dep-c842e491.js:41164:19)
      at normalizeUrl (file:///client/node_modules/vite/dist/node/chunks/dep-c842e491.js:38032:33)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async TransformContext.transform (file:///client/node_modules/vite/dist/node/chunks/dep-c842e491.js:38165:47)
      at async Object.transform (file:///client/node_modules/vite/dist/node/chunks/dep-c842e491.js:41421:30)
      at async loadAndTransform (file:///client/node_modules/vite/dist/node/chunks/dep-c842e491.js:37808:29)
```
As the error message says, the file you are trying to import in `client/src/context/TransactionContext.jsx`, `../utils/constants.js{,x}`, does not exist. The resolved path to this file is `client/src/utils/constants.js{,x}` and this indeed does not exist. Could you possibly mean to import the file at `client/utils/constants.js`?

**Reply by DevSammy22** [reply]
Yes. The file should be imported at client/utils/constants.js. Please, help me to resolve the issue.

**Reply by wongjn** [reply]
You could try editing the `import` statement in `client/src/context/TransactionContext.jsx` to point to the correct file:
```diff
 import { ethers } from "ethers";
-import { contractABI, contractAddress } from "../utils/constants";
+import { contractABI, contractAddress } from "../../utils/constants";

```

**Reply by DevSammy22** [reply]
I have restructured the folder and imported the statement to point to the correct line, but still showing nothing.
![image](https://user-images.githubusercontent.com/98836006/201483741-5673e39c-adba-4fc6-bd1a-049d0cbe00cf.png)

**Reply by wongjn** [reply]
Look for any more errors in the console. They should give you clues as to where the errors are and then you should be able to resolve them.

**Reply by DevSammy22** [reply]
This is my console. I did not see any error there.
![image](https://user-images.githubusercontent.com/98836006/201483913-7736c720-f557-4c32-931b-064288a78848.png)

**Reply by wongjn** [reply]
I've had another look at the repo and it seems like may wish to include your app code into your entry point to "load" it in to the page. If you're not sure how to do this, I would advise referring to React or Vite discussion areas, since this is a discussion area around Tailwind CSS and not a general discussion area.

**Reply by DevSammy22** [reply]
Thank you very much, Wong. Please, can you help me about this?

**Reply by wongjn** [reply]
I would advise referring to React or Vite discussion areas for this – it's not really pertaining to Tailwind!

**Reply by DevSammy22** [reply]
Thank you, @wongjn

**Reply by DevSammy22** [reply]
See the error I got in my console.

![image](https://user-images.githubusercontent.com/98836006/201485929-5eda351c-1af8-4c98-9d4c-87f1fea9630f.png)

**Reply by wongjn** [reply]
Consider reading the warning messages – they include URLs to documentation that you can read in an internet browser that should help guide you in rectify your configuration.

**Reply by DevSammy22** [reply]
I have rectified the configuration following the documentation, but I was getting no result positive.
